core/controlplane/config/templates/cluster.yaml

```diff
@@ -1214,6 +1214,11 @@ addons:
   metricsServer:
     enabled: false
 
+  # When set to true this configures security groups for prometheus between nodes.
+  # This includes the following ports: 10252, 10251, 10250, 9100, and 4194
+  prometheus: 
```

Comment:
Would you mind if I changed this to something like `node.exposeKubernetesSystemCompomentsPortsForPrometheus` so that it won't look like a general configuration section for Prometheus itself?

Reply:
Perfect. I wasn't sure exactly how to structure the config flag.